swabbie-core/src/main/kotlin/com/netflix/spinnaker/config/SwabbieProperties.kt

```diff
@@ -183,6 +184,14 @@ class Attribute {
   }
 }
 
+/**
+ * A simple configured rule
+ */
```

Comment:
For some reason I'm having trouble understanding this based on the name. Is `BasicRule` just a key/value match rule?